Formula/smimesign.rb

```diff
@@ -0,0 +1,22 @@
+class Smimesign < Formula
+  desc "S/MIME signing utility for use with Git"
+  homepage "https://github.com/github/smimesign"
+  url "https://github.com/github/smimesign/archive/0.0.3.tar.gz"
+  sha256 "69dfa16737c715679e067d0d929dd964790060b33981695d71bd6b047ae64e97"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/github/smimesign").install buildpath.children
+
+    cd "src/github.com/github/smimesign" do
+      system "go", "build", "-o", bin/"smimesign", "-ldflags", "-X main.versionString=#{version}"
```

Comment:
Does this download dependencies? How are they specified? (I am always unsure, pinning dependencies in go-based formulas is always a bit tricky…)

Reply:
This tool vendors its dependencies, so nothing is downloaded at build time.